lib/msal-node/src/cache/Storage.ts

```diff
@@ -245,7 +245,8 @@ export class Storage extends CacheManager {
 
         // read all keys
         Object.keys(cache).forEach(key => {
-            Object.keys(key).forEach(internalKey => {
+            // @ts-ignore
```

Comment:
The upcoming PR that addresses `storage` will get rid of this ts-ignore. Put this out to proceed with testing today.

Reply:
Addressed this.